### ISSUE TITLE
Add error assertion for test on GetTimestampAtHeight

### ISSUE THREAD

**Issue by colin-axner** [cosmos/ibc-go#5964]
## Summary

While reviewing #5806, I noticed that the test (which is moved to 02-client keeper tests) for `GetTimestampAtHeight` uses `expPass` instead of `expError`. This can be updated so the test asserts the expected error to be returned

Should be addressed after merging #5806 

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

**Comment by chandiniv1** [comment]
@colin-axner can I work on this issue?

**Comment by colin-axner** [comment]
Hi @chandiniv1, yes that would be wonderful! Can you wait until #5806 is merged into main? The test is moved around in that pr, so it will be easier to address this issue once that pr is merged

**Comment by chandiniv1** [comment]
@colin-axner yeah it's fine. I'll wait till that issue got merged
